spread.yaml

```diff
@@ -371,6 +380,23 @@ backends:
                   username: external
                   password: ubuntu
 
+    classic:
```

Comment:
should this be named external-classic or maybe desktop ?

Reply:
or classic-desktop ?

Reply:
Are those only desktop systems? Perhaps external-desktop then?

Reply:
classic-desktop sounds good, I don't like external because on external backends we don't build the current branch, we just use the system as it is